Makefile

```diff
@@ -71,13 +111,15 @@ push-aws-scanner:
 	@$(CONTAINER_ENGINE) push $(SCANNER_IMAGE):$(VERSION)
 	@$(CONTAINER_ENGINE) push $(SCANNER_IMAGE):$(IMAGE_TAG)
 
-start-dev: build
+
+# Development env based on Docker/Podman Compose tool
+start-dev:
 	@echo "### [Starting dev environment] ###"
-	@$(CONTAINER_ENGINE)-compose -f deploy/docker-compose/docker-compose.yaml up &
+	@$(CONTAINER_ENGINE)-compose -f $(DEPLOYMENTS_DIR)/docker-compose/docker-compose.yaml up &
 
 stop-dev:
 	@echo "### [Stopping dev environment] ###"
-	@$(CONTAINER_ENGINE)-compose -f deploy/docker-compose/docker-compose.yaml down
+	@$(CONTAINER_ENGINE)-compose -f $(DEPLOYMENTS_DIR)/docker-compose/docker-compose.yaml down
 
 help:
```

Comment:
WDYT if we add `help` as the default target (**fully optional**)?

```text
# Set the default target to "help"
.DEFAULT_GOAL := help
```

Reply:
Done